openmdao/devtools/tests/test_viewconns.py

```diff
@@ -1,22 +1,22 @@
 import unittest
 
-from openmdao.api import Problem
-from openmdao.test_suite.components.sellar import SellarNoDerivatives
-from openmdao.api import view_connections
 
 class TestSellarFeature(unittest.TestCase):
 
-    def test_sellar(self):
-        # Just tests Newton on Sellar with FD derivs.
+    # no output checking, just make sure no exceptions raised
+    # Just tests Newton on Sellar with FD derivs.
+    def test_feature_sellar(self):
+        from openmdao.api import Problem
+        from openmdao.test_suite.components.sellar import SellarNoDerivatives
+        from openmdao.api import view_connections
 
         prob = Problem()
         prob.model = SellarNoDerivatives()
 
         prob.setup()
         prob.final_setup()
 
-        # no output checking, just make sure no exceptions raised
-        view_connections(prob, show_browser=False)
+        view_connections(prob, outfile= "sellar_connections.html", show_browser=False)
```

Comment:
Should probably put the `@use_tempdirs` decorator on this test so that it cleans up this temporary file,